packages/react/src/provider/use-open-feature-client.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Context } from './context';
+
+/**
+ * Get the {@link Client} instance for this OpenFeatureProvider context.
+ * Note that the provider to which this is bound is determined by the OpenFeatureProvider's domain.
+ * @returns Client
+ */
+export function useOpenFeatureClient() {
```

Comment:
Extracted.

@lukas-reining you originally wrote this, and it certainly makes sense internally, but I also think it's useful to consumers. Do you think we should continue to expose it? I think it could be useful.

Reply:
> Do you think we should continue to expose it? I think it could be useful.

Yes I think noting speaks against it.
There might be a reason to just get the client directly, e.g. you do not have a suspense boundary or want to use a flag value in a callback that should evaluate the value on call (sure this can be done differently but it might be convenient to just pass the client.
And I guess it should also not be a problem regarding encapsulation or so. We also have this in Nest.js SDK.